package.json

```diff
@@ -1,9 +1,9 @@
 {
-	"name": "mono-debug",
-	"displayName": "Mono Debug",
-	"version": "0.15.7",
+	"name": "xamarin",
+	"displayName": "Xamarin",
```

Comment:
I should probably flip this back before we merge into microsoft's repo?

Reply:
It raises a good question. We may want to make this it's own extension tbh.

Reply:
we probably do want to consider this a new extension